arch/platform/native/Makefile.native

```diff
@@ -22,6 +22,8 @@ endif
 
 CONTIKI_SOURCEFILES += $(CONTIKI_TARGET_SOURCEFILES)
 
+CFLAGS += -Wno-stringop-truncation
```

Comment:
why not place it to Makefile.include in
```
ifeq ($(WERROR),1)
 LDFLAGS += $(LDFLAGS_WERROR)
endif
```
?

Reply:
imho, comment about why this warning need suprees would great

Reply:
These are good suggestions, but in my mind this is just a quick fix for the release that will be reverted later. The strcnpy usage should be fixed in the code, which I do for the native platform in #1437.